README.md

```diff
@@ -180,7 +180,7 @@ This plugin creates device plugin endpoints based on the configurations given in
 |     Field      | Required |        Description        |                      Type - Accepted values                       |                                      Example/Accepted values                                       |
 |----------------|----------|---------------------------|-------------------------------------------------------------------|----------------------------------------------------------------------------------------------------|
 | "resourceName" | Yes      | Endpoint resource name    | string - must be unique and should not contain special characters | "sriov_net_A"                                                                                      |
-| "selectors"    | No       | A map of device selectors | Each selector is a map of string list.                            | "vendors": ["8086"], "devices": ["154c", "10ed"], "drivers": ["vfio-pci"], "pfNames": ["enp2s2f0"] |
+| "selectors"    | No       | A map of device selectors | Each selector is a map of string list.                            | "vendors": ["8086"], "devices": ["154c", "10ed"], "drivers": ["vfio-pci"], "pfNames": ["enp2s2f0"], "linkTypes": ["ether"] |
```

Comment:
what about adding an example config in above `resourceList`?
Q: Is there any other `linkType` other than `ether`, `infiniband`?

Reply:
Sure, will add a new entry .

in regards to link types, netlink go lib defines the string values in: https://github.com/vishvananda/netlink/blob/b1e98597921720cee4147d6476110fc8fc56072d/nl/nl_linux.go#L111

which are derived from: https://github.com/torvalds/linux/blob/master/include/uapi/linux/if_arp.h

Reply:
Sure, will add a new entry  in the resourceList above.

in regards to link types, netlink go lib defines the string values in: https://github.com/vishvananda/netlink/blob/b1e98597921720cee4147d6476110fc8fc56072d/nl/nl_linux.go#L111

which are derived from: https://github.com/torvalds/linux/blob/master/include/uapi/linux/if_arp.h